src/couch_httpd.erl

```diff
@@ -30,6 +30,7 @@
 -export([send_json/2,send_json/3,send_json/4,last_chunk/1,parse_multipart_request/3]).
 -export([accepted_encodings/1,handle_request_int/5,validate_referer/1,validate_ctype/2]).
 -export([http_1_0_keep_alive/2]).
+-export([add_default_headers_if_needed/2, add_header_if_needed/2]).
```

Comment:
I am unsure if I get this one, do you mean adding something like the following at the end of this module? 

``` erlang
-ifdef(TEST).
add_default_headers_test() -> something.
-endif.
```
If yes, sure, will do.